wifiphisher/pywifiphisher.py

```diff
@@ -39,6 +39,12 @@ def parse_args():
     # Create the arguments
     parser = argparse.ArgumentParser()
 
+    # Interface selection
+    parser.add_argument(
+        "-i",
+        "--interface",
+        help=("Choose an interface that is connected on the Internet" +
+              "Example: -i wlan1"))
```

Comment:
@sophron, A little confused about the help message. Isn't this option used for an interface supporting both monitor and AP mode, then this interface will not be used to connect internet, right?

Reply:
Correct. I need to edit.